code-challenges.rb

```diff
@@ -11,23 +11,81 @@
 # Expected output: '221 is odd'
 
 
+# created a method named odd_or_even
+def odd_or_even(num)
+# compose an of statement to determine whether the argument is odd is or even.
+    if num % 2 == 0
+        # if the number is even, the implicit return is the following
+        "#{num} is even"
+    else
+        # if the number is odd, the implicit return is the following
+        "#{num} is odd"
+    # end for the if block
+    end
+# end for the method
+end
+
+p odd_or_even("#{num1}")
+p odd_or_even("#{num2}")
+p odd_or_even("#{num3}")
```

Comment:
This is an interesting approach with your inputs. I wonder if stringifying the inputs would have any side effects.